tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -333,6 +384,30 @@ inline uint32_t getNumBaseClasses(QualType type) {
     return cxxDecl->getNumBases();
   return 0;
 }
+
+/// Returns the appropriate storage class for an extern variable of the given
+/// type.
+spv::StorageClass getStorageClassForExternVar(QualType type,
+                                              bool hasGroupsharedAttr) {
+  // For CS groupshared variables
+  if (hasGroupsharedAttr)
+    return spv::StorageClass::Workgroup;
+
+  if (isAKindOfStructuredOrByteBuffer(type))
+    return spv::StorageClass::Uniform;
```

Comment:
I do not clearly remember how we handled spirv1.3 or above but I want to doublecheck it will be ok to set it as only `Uniform` storage class, not `StorageBuffer`.